source/administration/object-management.rst

```diff
@@ -59,16 +71,48 @@ However, hardware and network conditions may show performance impacts with large
 For a deeper discussion on the benefits of limiting prefix contents, see the article on :s3-docs:`optimizing S3 performance <optimizing-performance.html>`.
 
 Object Versioning
------------------
+----------------- 
 
 MinIO supports keeping multiple "versions" of an object in a single bucket. 
-Write operations which would normally overwrite an existing object instead result in the creation of a new versioned object. 
 
 .. image:: /images/retention/minio-versioning-multiple-versions.svg
    :alt: Object with Multiple Versions
    :align: center
 
-MinIO versioning protects from unintended overwrites and deletions while providing support for "undoing" a write operation.
+The specific client behavior on write, list, get, or delete operations on a bucket depends on the versioning state of that bucket:
+
+.. list-table::
+   :stub-columns: 1
+   :header-rows: 1
+   :widths: 20 40 40
+   :width: 100%
+
+   * - Operation
+     - Versioning Enabled
+     - Versioning Disabled | Suspended
+
+   * - ``PUT`` (Write)
+     - Create a new full version of the object as the "latest" and assign a unique version ID
+     - Create the object with overwrite on namespace match.
+
+   * - ``GET`` (Read)
+     - Retrieve the latest version of the object by default
+
+       Supports retrieving a specific object by version ID.
```

Comment:
Maybe 
> Supports retrieving any object version by version ID

Reply:
:thinking: i'm trying to avoid the double 'version" here. But I see the concern, so meh.